src/content/en/shows/http203/podcast/corb-cast-switch-face.md

```diff
@@ -0,0 +1,42 @@
+project_path: /web/_project.yaml
+book_path: /web/shows/_book.yaml
+description: Also spices, screen-touchers, and lasers.
```

Comment:
Is this the part that gets pulled into the feed for the other platform? If so, it might need to be a bit more descriptive, no? ^^

Reply:
Both the title and description go in. But yeah, I'd like the full page content to go into the feed, but I just haven't had time to look at it.